### PR TITLE
Alert: Endpoints Migration From Ethercluster to Rivet

### PR DESCRIPTION
**Pull request by DonaldMcIntyre**
This is an alert for all the ecosystem to migrate from Ethercluster to Rivet as announced by the ETC Cooperative.

It is for 2023-01-04.

### CODE REVIEW

**Comment by gitr0n1n** [comment]
A draft, not ready for review with less than 24 hours as a post date?

@DonaldMcIntyre please provide longer review times for these articles. I see six incomplete drafts and one PR ready for review. All with pressing dates.

Please finish these PRs and then schedule posting dates. You're putting reviewers in a rushed situation with this current approach. Thanks in advance. We shouldn't feel a need to check more than once a week imo.

Not a big deal and not negative, we just need to iron out these work flow issues. 👍

**Comment by DonaldMcIntyre** [comment]
> A draft, not ready for review with less than 24 hours as a post date?
> 
> @DonaldMcIntyre please provide longer review times for these articles. I see six incomplete drafts and one PR ready for review. All with pressing dates.
> 
> Please finish these PRs and then schedule posting dates. You're putting reviewers in a rushed situation with this current approach. Thanks in advance. We shouldn't feel a need to check more than once a week imo.
> 
> Not a big deal and not negative, we just need to iron out these work flow issues. 👍

Thank you Ronin, sorry for this one, it was a last minute addition to replace the one that we dropped for tomorrow.

In November and December the flow went well, I think these days it's slow to review because of the holidays and people are getting back to work on these things. Nevertheless, we have to provide the PRs for review several days in advance not in the last minute. 

Mind that the 6 that are pending are for all of January, they are not all urgent. The key is to have the Chinese translation as soon as possible so the PR is out of draft and ready for review many days in advance.

Also, the flow is only 3 articles per week, it doesn't seem too much for several admins to process. Again, sorry for this one in particular which was, indeed, a last minute addition. We can just wait until admins have time to process it.

**Review by w1g0 — CHANGES_REQUESTED** [review]
The Rivet Discord invite link is invalid for the English version as well as for the Chinese version.

**Review by w1g0 — DISMISSED** [review]

**Review by bobsummerwill — CHANGES_REQUESTED** [review]
I think this article needs toning down to just expressing the basic facts, and referring back to the article on the ETC Coop site.

It is inappropriate, for example, to have the "sell" from Rivet here or about Coop's history of providing this service, IMHO.

There are other people providing endpoints too.

**Comment by w1g0** [comment]
> I think this article needs toning down to just expressing the basic facts, and referring back to the article on the ETC Coop site.
> 
> It is inappropriate, for example, to have the "sell" from Rivet here or about Coop's history of providing this service, IMHO.
> 
> There are other people providing endpoints too.

The article is clearly addressed to people that currently are using EtherCluster to migrate to Rivet, so I see no issue there but I understand your POV and am fine with it either way.

**Comment by bobsummerwill** [comment]
We should have no need to rush this one.

The transition is happening over the next 6 months+

**Comment by DonaldMcIntyre** [comment]
> I think this article needs toning down to just expressing the basic facts, and referring back to the article on the ETC Coop site.
> 
> It is inappropriate, for example, to have the "sell" from Rivet here or about Coop's history of providing this service, IMHO.
> 
> There are other people providing endpoints too.

I will trim the ETC Coop's and Rivet's sell segments and refer to the Coop's original post, but I will leave the sense of urgency to make the migration and the datapoints of the endpoints.

Will change it today.

cc @w1g0

**Comment by DonaldMcIntyre** [comment]
@w1g0 @bobsummerwill  Changes done! It would be great if you could approve it so we publish it, thank you!

File: content/blog/2023-01-04-alert-migrate-your-etc-endpoints-from-ethercluster-to-rivet-by-july-1st-2023/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+The migration may be done immediately to the following RPC endpoint:
+
+[https://etc.rivet.link](https://etc.rivet.link)
+
+If you are a wallet service, node operator, mining pool, dapp developer, or any other kind of user of the Ethercluster endpoints, please migrate as soon as possible to avoid interruptions.
+
+The last day for the current Ethercluster rpc endpoint is July 1st of 2023.
+
+## Summary
```
Comment (bobsummerwill) [inline content/blog/2023-01-04-alert-migrate-your-etc-endpoints-from-ethercluster-to-rivet-by-july-1st-2023/index.md:35]: I think you can omit the Summary section at the end.  It is redundant, especially with the article now being so short.
Just keep the WARNING AT THE END, but not the Summary heading or body.

Also worth saying that alternative endpoints are also available.   See /network/endpoints/ for more details.

**Review by bobsummerwill — CHANGES_REQUESTED** [review]

File: content/blog/2023-01-04-alert-migrate-your-etc-endpoints-from-ethercluster-to-rivet-by-july-1st-2023/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+[https://ethercluster.com/etc](https://ethercluster.com/etc)
+
+The migration may be done immediately to the following RPC endpoint:
+
+[https://etc.rivet.link](https://etc.rivet.link)
+
+If you are a wallet service, node operator, mining pool, dapp developer, or any other kind of user of the Ethercluster endpoints, please migrate as soon as possible to avoid interruptions.
+
+The last day for the current Ethercluster rpc endpoint is July 1st of 2023.
```
Comment (TheCrowbill) [inline content/blog/2023-01-04-alert-migrate-your-etc-endpoints-from-ethercluster-to-rivet-by-july-1st-2023/index.md:33]:
```suggestion
The last day for the current Ethercluster RPC endpoint is July 1st of 2023.
```

**Review by TheCrowbill — APPROVED** [review]
After the requested changes, LGTM.

**Review by bobsummerwill — COMMENTED** [review]
English text looks good.
We need to wait on the Chinese translation, and then I am good to merge.

**Review by TheCrowbill — APPROVED** [review]

**Comment by DonaldMcIntyre** [comment]
> English text looks good. We need to wait on the Chinese translation, and then I am good to merge.

Chinese version is updated.

**Review by bobsummerwill — APPROVED** [review]
LTGM, but please do remember to squash the commits on merge.  Messy history with all the iteration.  Thanks!

**Comment by DonaldMcIntyre** [comment]
> LTGM, but please do remember to squash the commits on merge.  Messy history with all the iteration.  Thanks!

Squished and merged, thank you!

**Pull request merged**